awscognito/cognito_users.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=global-statement,redefined-outer-name
+""" Script used to create|delete AWS Cognito user """
+import argparse
+import sys
+
+
+def create_user(client, profile, user):
+    """ Creates a new user in the specified user pool """
+    response = client.admin_create_user(
+        UserPoolId=profile.pool_id,
+        Username=user.email,
+        UserAttributes=[
+            {"Name": "email", "Value": user.email},
+            {"Name": "custom:name", "Value": user.name},
+        ],
+        ValidationData=[{"Name": "string", "Value": "string"},],
+        MessageAction="RESEND",
+        DesiredDeliveryMediums=["EMAIL"],
+    )
+    return response
+
+
+def delete_user(client, profile, user):
```

Comment:
Instead of deleting users, maybe it is better to "disable" users so we can manually enable them if needed.
(Just in case we accidentally delete all users.)

Reply:
I will add 'disable_user' for production use and keep this one for "internal" testing (do not want to leave too many dummy accounts)